src/pipecat/services/ai_services.py

```diff
@@ -137,6 +141,13 @@ async def process_generator(self, generator: AsyncGenerator[Frame | None, None])
                     await self.push_frame(f)
 
 
+@dataclass
+class FunctionCallEntry:
+    function_name: Optional[str]
+    callback: Any  # TODO(aleix): add proper typing.
```

Comment:
nit: `callback` is the function itself, right? any reason it's called `callback`? (to me, that makes it sound like something that happens on completion). But maybe this is standard naming in Python or this project that I'm just ignorant of, in which case ignore me!

Reply:
It could be `function`.

Reply:
Well, callbacks are just functions called by other functions, so I think it's a good term.

Reply:
I guess it's an old school term in this sense.